### ISSUE TITLE
检测训练精度很高，c++检测推理精度很低

### ISSUE THREAD

**Issue by ronaldpan** [PaddlePaddle/PaddleOCR#2584]
检测训练的配置文件
Global:
  debug: false
  use_gpu: true
  epoch_num: 1200
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/ch_db_mv3/
  save_epoch_step: 1200
  eval_batch_step:
  - 3000
  - 2000
  load_static_weights: true
  cal_metric_during_train: false
  pretrained_model: ./pretrain_models/MobileNetV3_large_x0_5_pretrained
  checkpoints: null
  save_inference_dir: null
  use_visualdl: false
  infer_img: doc/imgs_en/img_10.jpg
  save_res_path: ./output/det_db/predicts_db.txt
  pretrain_weights: ./pretrain_models/MobileNetV3_large_x0_5_pretrained/
  distributed: false
Architecture:
  model_type: det
  algorithm: DB
  Transform: null
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: large
    disable_se: true
  Neck:
    name: DBFPN
    out_channels: 96
  Head:
    name: DBHead
    k: 50
Loss:
  name: DBLoss
  balance_loss: true
  main_loss_type: DiceLoss
  alpha: 5
  beta: 10
  ohem_ratio: 3
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 2
  regularizer:
    name: L2
    factor: 0
PostProcess:
  name: DBPostProcess
  thresh: 0.7
  box_thresh: 0.2
  max_candidates: 1000
  unclip_ratio: 1.2
Metric:
  name: DetMetric
  main_indicator: hmean
Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/36e01ocr_det/
    label_file_list:
    - ./train_data/36e01ocr_det/train_label.txt
    ratio_list:
    - 1.0
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - IaaAugment:
        augmenter_args:
        - type: Fliplr
          args:
            p: 0.5
        - type: Affine
          args:
            rotate:
            - -10
            - 10
        - type: Resize
          args:
            size:
            - 0.5
            - 3
    - EastRandomCropData:
        size:
        - 960
        - 960
        max_tries: 50
        keep_ratio: true
    - MakeBorderMap:
        shrink_ratio: 0.4
        thresh_min: 0.3
        thresh_max: 0.7

c++中检测部分的配置如下：
#det config
max_side_len  960
det_db_thresh  0.7
det_db_box_thresh  0.2
det_db_unclip_ratio 1.2
det_model_dir  ./inference/ch_ppocr_mobile_v2.0_det_infer/
c++检测结果完全不对。请问是哪里出问题了，配置文件应该怎么修改？

**Comment by LDOUBLEV** [comment]
你在python端测试导出的inference model预测是不是对的，用tools/infer/predict_det.py这个脚本
python3.7 tools/infer/predict_det.py --det_model_dir="./your inference model/"  --image_dir=./image_dir/  --use_gpu=True

**Comment by ronaldpan** [comment]
在python端测试导出的inference model预测，结果是对的。

**Comment by ronaldpan** [comment]
在PPOCRLabel.py中加载我导出的inference model，结果一部分是对的。
        self.ocr = PaddleOCR(
det_model_dir='D:/PaddleOCR/output/ch_db_mv3/',
det_db_thresh = 0.7,
det_db_box_thresh = 0.2,
det_db_unclip_ratio = 1.2,
use_pdserving=False, 
use_angle_cls=True, 
det=True, 
cls=True, 
use_gpu=True, lang=lang)
